coordinator-server/event-client/actionSniffer.js

```diff
@@ -45,8 +46,8 @@ const EventHandlers = {
             height: rectangle.bottom - rectangle.top
         };
         const position = {
-            x: event.pageX - rectangle.left,
-            y: event.pageY - rectangle.top
+            x: Math.max(event.pageX - rectangle.left - window.pageXOffset, 0),
```

Comment:
would reorder these subtractions, so that rectangle.left is at the end, this makes it more clear that we are removing the scroll and the the item